backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java

```diff
@@ -48,14 +50,13 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
       CameraController cameraController = ServerController.getInstance().getCameraController();
       Camera camera = cameraController.getCameraById(Integer.parseInt(camID));
       String tags = request.getParameter("tags");
-      List<String> tagList = null;
-      
+
+      List<String> tagList = new ArrayList<String>();
```

Comment:
I think the `String` between diamonds can be removed because the compiler will know it is a `String` type.